db/migrate/20180810033347_create_sierra_code_zcode_joins.rb

```diff
@@ -0,0 +1,10 @@
+class CreateSierraCodeZcodeJoins < ActiveRecord::Migration
```

Comment:
The name of migration and class are mis-leading to me. This is technically not a join table if both values are being derived from the same table without a join.

Reply:
Sounds good, @nnat425, I made the change from "joins" to "matches" as we described.